samples/python/hello_classification/hello_classification.py

```diff
@@ -41,49 +40,70 @@ def main():
         log.error('Sample supports only single output topologies')
         return -1
 
-# --------------------------- Step 3. Set up input --------------------------------------------------------------------
-    # Read input image
-    image = cv2.imread(image_path)
-    # Add N dimension
-    input_tensor = np.expand_dims(image, 0)
-
-# --------------------------- Step 4. Apply preprocessing -------------------------------------------------------------
-    ppp = PrePostProcessor(model)
-
-    _, h, w, _ = input_tensor.shape
-
-    # 1) Set input tensor information:
-    # - input() provides information about a single model input
-    # - precision of tensor is supposed to be 'u8'
-    # - layout of data is 'NHWC'
-    # - set static spatial dimensions to input tensor to resize from
-    ppp.input().tensor() \
-        .set_element_type(Type.u8) \
-        .set_layout(Layout('NHWC')) \
-        .set_spatial_static_shape(h, w)  # noqa: ECE001, N400
-
-    # 2) Adding explicit preprocessing steps:
-    # - apply linear resize from tensor spatial dims to model spatial dims
-    ppp.input().preprocess().resize(ResizeAlgorithm.RESIZE_LINEAR)
-
-    # 3) Here we suppose model has 'NCHW' layout for input
+    if (device_name != 'MYRIAD'):
+        # --------------------------- Step 3. Set up input --------------------------------------------------------------------
+        # Read input image
+        image = cv2.imread(image_path)
+        # Add N dimension
+        input_tensor = np.expand_dims(image, 0)
+
+        # --------------------------- Step 4. Apply preprocessing -------------------------------------------------------------
+        ppp = PrePostProcessor(model)
+
+        _, h, w, _ = input_tensor.shape
+
+        # 1) Set input tensor information:
+        # - input() provides information about a single model input
+        # - precision of tensor is supposed to be 'u8'
+        # - layout of data is 'NHWC'
+        # - set static spatial dimensions to input tensor to resize from
+        ppp.input().tensor() \
+            .set_element_type(Type.u8) \
+            .set_layout(Layout('NHWC')) \
+            .set_spatial_static_shape(h, w)  # noqa: ECE001, N400
+
+        # 2) Adding explicit preprocessing steps:
+        # - apply linear resize from tensor spatial dims to model spatial dims
+        ppp.input().preprocess().resize(ResizeAlgorithm.RESIZE_LINEAR)
```

Comment:
@DariaMityagina I suppose MYRIAD plugin must call transformation from here https://github.com/openvinotoolkit/openvino/pull/8204 could you please have a look?

Reply:
I will take a look. Thank you!

Reply:
@DariaMityagina have you addressed @ilya-lavrenov comment?

Reply:
Yes, I have.